WebApp/src/test/java/ev/projects/webapp/utils/EntitiesCreator.java

```diff
@@ -0,0 +1,40 @@
+package ev.projects.webapp.utils;
+
+import ev.projects.models.Case;
+import ev.projects.models.Document;
+import ev.projects.models.User;
+
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+
+public class EntitiesCreator {
```

Comment:
Instead of this use a factory design pattern and create 2 different factories, one for Case, another for document.

Reply:
Will do. Will be back at this tomorrow.

Reply:
Suggestion fulfilled in cb8c716.

Reply:
Also, going with your idea further, I think document factory should be split in two factory methods like in 8ebc906, to prevent ifs. Maybe this solution is better than the previous one?

Reply:
Yes, I think 2 methods is better.